packages/astro/src/core/config.ts

```diff
@@ -343,16 +344,11 @@ export async function validateConfig(
 		_ctx: {
 			pageExtensions: ['.astro', '.md', '.html'],
 			scripts: [],
-			renderers: [],
+			renderers: [jsxRenderer],
```

Comment:
why?

Reply:
Previously the Astro JSX renderer was only enabled if `@astrojs/mdx` was included. Now our default `.md` compilation relies on this renderer for Layouts, too.

I don't love that this puts _every_ project in a multiple-JSX renderer world, but I can't think of another option at the moment.